pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/BitSlicedRangeIndexReader.java

```diff
@@ -23,20 +23,25 @@
 import javax.annotation.Nullable;
 import org.apache.pinot.segment.local.segment.creator.impl.inv.BitSlicedRangeIndexCreator;
 import org.apache.pinot.segment.local.utils.FPOrdering;
+import org.apache.pinot.segment.spi.ColumnMetadata;
 import org.apache.pinot.segment.spi.index.reader.RangeIndexReader;
 import org.apache.pinot.segment.spi.memory.PinotDataBuffer;
 import org.roaringbitmap.RangeBitmap;
 import org.roaringbitmap.RoaringBitmap;
 import org.roaringbitmap.buffer.ImmutableRoaringBitmap;
+import org.roaringbitmap.buffer.MutableRoaringBitmap;
 
 
 public class BitSlicedRangeIndexReader implements RangeIndexReader<ImmutableRoaringBitmap> {
 
   private final PinotDataBuffer _dataBuffer;
   private final long _offset;
   private final long _min;
+  private final long _max;
+  private final long _maxRid;
```

Comment:
(nit) We usually use `_numDocs`. It is currently exclusive, which can possibly cause confusion

Reply:
I will change this.